tasks/RedHat.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: "Install epel repo"
```

Comment:
I tested on a clean RHEL 7.5, and there's no `epel-release` package available:
<pre>
[root@localhost ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.5 (Maipo)

[root@localhost ~]# sudo yum install epel-release
Loaded plugins: product-id, search-disabled-repos, subscription-manager
No package epel-release available.
Error: Nothing to do
</pre>


To add the epel-release repo on RHEL 7.5, it is needed to install the  https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm package first, for example:

<pre>
    - name: "Install EPEL repository on RHEL"
      become: "yes"
      yum:
        name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
        state: present
      when: ansible_distribution == 'RedHat'

    - name: "Add rhel-*-optional-rpms repo on RHEL"
      become: "yes"
      command: subscription-manager repos --enable "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"
      when: ansible_distribution == 'RedHat'
</pre>

NOTE: about the second repo added, on RHEL 7 it is recommended to also enable the optional and extras repositories since EPEL packages may depend on packages from these repositories

More info at:
https://aws.amazon.com/premiumsupport/knowledge-center/ec2-enable-epel/
https://fedoraproject.org/wiki/EPEL